tobac/tests/test_utils.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import tobac.utils as tb_utils
+
+
+def test_spectral_filtering():
+    """Testing tobac.utils.spectral_filtering with random test data."""
+
+    # generate 3D data with random values
+    random_data = np.random.rand(100, 300, 500)
```

Comment:
Should we set a seed for this, or will any random values work?

Reply:
I have resolved this by seeding a random wave signal with a wavelength outside of the set range for filtering. This can then be used to test the desired behavour of the spectral filtering: basically to remove this signal as it is not in the desired wavelength range.